docs/dockermachine.md

```diff
@@ -0,0 +1,449 @@
+INFO[0007] Waiting for VM to start...
+INFO[0038] "dev" has been created and is now the active machine. To point Docker at this machine, run: export DOCKER_HOST=$(machine url) DOCKER_AUTH=identity
+```
+
+You can see the machine you have created by running the `machine ls` command 
+again:
+
+```
+$ machine ls
+NAME      ACTIVE   DRIVER       STATE     URL
+dev       *        virtualbox   Running   tcp://192.168.99.100:2375
+```
+
+The `*` next to `dev` indicates that it is the active host.
+
+Next, as noted in the output of the `machine create` command, we have to tell 
+Docker to talk to that machine directly by setting the `DOCKER_HOST` 
+and `DOCKER_AUTH` environment variables:
+
+```
+$ export DOCKER_HOST=$(machine url) DOCKER_AUTH=identity
+```
+
+You can now run Docker commands on this host:
+
+```
+$ docker run busybox echo hello world
+Unable to find image 'busybox' locally
+Pulling repository busybox
+e72ac664f4f0: Download complete
+511136ea3c5a: Download complete
+df7546f9f060: Download complete
+e433a6c5b276: Download complete
+hello world
+```
+
+Any exposed ports are available on the Docker host’s IP address, which you can 
+get using the `machine ip` command:
+
+```
+$ machine ip
+192.168.99.100
+```
+
+Now you can manage as many local VMs running Docker as you please- just run 
+`machine create` again.
+
+If you are finished using a host, you can stop it with `docker stop` and start 
+it again with `docker start`:
+
+```
+$ machine stop
+$ machine start
+```
+
+If they aren't passed any arguments, commands such as `machine stop` will run 
+against the active host (in this case, the VirtualBox VM).  You can also specify 
+a host to run a command against as an argument.  For instance, you could also 
+have written:
+
+```
+$ machine stop dev
+$ machine start dev
+```
+
+## Using Docker Machine with a cloud provider
+
+One of the nice things about `machine` is that it provides several “drivers” 
+which let you use the same interface to create hosts on many different cloud 
+platforms.  This is accomplished by using the `machine create` command with the
+ `--driver` flag.  Here we will be demonstrating the 
+[Digital Ocean](https://digitalocean.com) driver (called `digitalocean`), but 
+there are drivers included for several providers including Amazon Web Services, 
+Google Compute Engine, and Microsoft Azure.
+
+Usually it is required that you pass account verification credentials for these 
+providers as flags to `machine create`.  These flags are unique for each driver.  
+For instance, to pass a Digital Ocean access token you use the 
+`--digitalocean-access-token` flag.
+
+Let's take a look at how to do this.
+
+To generate your access token:
+
+1. Go to the Digital Ocean administrator panel and click on "Apps and API" in 
+the side panel.
+2. Click on "Generate New Token".
+3. Give the token a clever name (e.g. "machine"), make sure the "Write" checkbox 
+is checked, and click on "Generate Token".
```

Comment:
can this token be used from many computers? add warning about keeping it safe, is it stored in a secure fashion, as anyone that gets it can then use my account to make a bitcoin farm....